components/inputs/test/input-number.vdiff.js

```diff
@@ -4,6 +4,28 @@ import { clickAt, expect, fixture, focusElem, html, oneEvent, sendKeysElem } fro
 
 const requiredFixture = html`<d2l-input-number label="Number" required></d2l-input-number>`;
 const simpleFixture = html`<d2l-input-number label="Number"></d2l-input-number>`;
+const inlineHelpComponents = {
```

Comment:
Nit: let's be consistent with the variable naming... let's call this `inlineHelpFixtures`.